src/renderer/services/binance/service.ts

```diff
@@ -335,63 +351,63 @@ const assetTxs$: AssetTxsPageLD = Rx.combineLatest([
  * If a client is not available (e.g. by removing keystore), it returns `None`
  *
  */
-const explorerUrl$: Observable<O.Option<string>> = client$.pipe(
+export const explorerUrl$: Observable<O.Option<string>> = client$.pipe(
   map(FP.pipe(O.map((client) => client.getExplorerUrl()))),
   shareReplay(1)
 )
 
+const getExplorerTxUrl$: Observable<O.Option<(ts: string) => string>> = client$.pipe(
+  map(FP.pipe(O.map((client) => client.getExplorerTxUrl))),
+  shareReplay(1)
+)
+
 // `TriggerStream` to reload `Fees`
 const { stream$: reloadFees$, trigger: reloadFees } = triggerStream()
 
-/**
- * Observable to load transaction fees from Binance API endpoint
- * If client is not available, it returns an `initial` state
- */
-const loadFees$ = (client: BinanceClient): FeesLD =>
-  Rx.from(client.getFees()).pipe(
-    map(RD.success),
-    catchError((error) => Rx.of(RD.failure(error))),
-    startWith(RD.pending),
-    retry(BINANCE_MAX_RETRY)
-  )
-
 /**
  * Transaction fees
  * If a client is not available, it returns `None`
  */
-const fees$: FeesLD = Rx.combineLatest([reloadFees$, client$]).pipe(
-  mergeMap(([_, oClient]) =>
+const transferFees$: Observable<TransferFeesRD> = Rx.combineLatest([reloadFees$, client$]).pipe(
+  switchMap(([_, oClient]) =>
     FP.pipe(
       // client and asset has to be available
       oClient,
       // ignore all assets from other chains than BNB
       O.fold(
-        () => Rx.of(RD.initial),
-        (client) => loadFees$(client)
+        () => Rx.EMPTY,
+        (client) => Rx.from(Promise.all([client.getFees(), client.getMultiSendFees()]))
```

Comment:
This will end up in two requests to the same endpoint behind the scenes. Currently that's an issue of `xchain-binance` (not of this PR), but we should fix it or at least create an issue about it at `xchainjs`. 

- Request one: https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-binance/src/client.ts#L391
- Request two: https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-binance/src/client.ts#L415

Reply:
agree. not sure what we should to do exactly. Do you suppose to make the only call for `getFees` and use it to provide data to the `single/multi` fees ?

Reply:
I would suppose to add another method to `xchain-binance`:

```ts
getSingleAndMultiFees(): Promise<{single: Fees, multi: Fees}>
```

However, I don't know if we still need `multi` Fees in ASGDX...?

Reply:
^ https://github.com/xchainjs/xchainjs-lib/issues/95